pkg/rbd/nodeserver.go

```diff
@@ -538,6 +539,48 @@ func (ns *NodeServer) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstag
 	return &csi.NodeUnstageVolumeResponse{}, nil
 }
 
+func (ns *NodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolumeRequest) (*csi.NodeExpandVolumeResponse, error) {
+
+	volumeID := req.GetVolumeId()
```

Comment:
question: do we need to validate volumeID?

Reply:
may be not required, but should be ok too.

Reply:
This additional check maybe required, as at present we have no other checks in this code that makes sure we are dealing with an RBD VolID and thus volume. Any plugins volume can be resized with this request, and prevention from erroneous CO requests will help avoid issues.

Reply:
ID validation is done, as there is no secret in this request we can not connect to the backend and validate it. More or less, this may be a rare or corner case which can be checked later too.